HW-11/homework11(1).py

```diff
@@ -0,0 +1,20 @@
+import json
+from datetime import datetime
+
+
+def MyDecFunc(func):
+    def wrap():
+        time = datetime.now()
+        text = f'Open function {func} \n Time: {time}'
+        with open(r'C:\HW\HomeWork\HW-11\file_dec', 'x') as file_dec:
+            text = json.dumps(text)
```

Comment:
Your text is not a JSON-like object, there is no need to use `json` here. You can use `json` to transform dict or list to `JSON` text, but for strings this is pointless